starport/services/scaffolder/message.go

```diff
@@ -13,7 +11,13 @@ import (
 )
 
 // AddMessage adds a new message to scaffolded app
-func (s *Scaffolder) AddMessage(moduleName string, msgName string, msgDesc string, fields []string, resFields []string) error {
+func (s *Scaffolder) AddMessage(
+	moduleName string,
+	msgName string,
+	msgDesc string,
+	fields []string,
+	resFields []string,
+) error {
```

Comment:
nit: it is a common practice to define type only once for all arguments that share it, e.g. "func(first, second string, firstArr, secondArr []string)", it will also make it short enough to fit on a single line